src/apps/Settings/Settings.js

```diff
@@ -14,10 +15,61 @@ import observeCache from '../../components/HOC/observeCache'
 import EtherscanLink from '../../components/Etherscan/EtherscanLink'
 import provideNetwork from '../../context/provideNetwork'
 import { compose } from '../../utils'
+import { getWeb3 } from '../../web3-utils'
+import { web3Providers, network, appIds } from '../../environment'
+import airdrop from '../../airdrop'
 
 // const AVAILABLE_CURRENCIES = ['USD', 'EUR', 'GBP', 'CAD', 'RMB', 'JPY']
 const AVAILABLE_CURRENCIES = ['USD'] // Only use USD for now
 
+const TEST_TOKENS = [
+  {
+    address: '0x0d5263b7969144a852d58505602f630f9b20239d',
+    symbol: 'ANT',
+    name: 'Aragon',
+  },
+  {
+    address: '0x6142214d83670226872d51e935fb57bec8832a60',
+    symbol: 'MANA',
+    name: 'Decentraland',
+  },
+  {
+    address: '0x1e1cab55639f67e70973586527ec1dfdaf9bf764',
+    symbol: 'BCC',
+    name: 'Bitconnect',
+  },
+  {
+    address: '0x5e381afb0104d374f1f3ccde5ba7fe8f5b8af0e6',
+    symbol: 'SPANK',
+    name: 'Spankchain',
+  },
+  {
+    address: '0xa53899a7eb70b309f05f8fdb344cdc8c8f272abe',
+    symbol: 'SNT',
+    name: 'Status',
+  },
+  {
+    address: '0x5b2fdbba47e8ae35b9d6f8e1480703334f48b96c',
+    symbol: 'DNT',
+    name: 'District0x',
+  },
+  /*{
```

Comment:
Which one is this?

Reply:
Not sure, there is no symbol / name on the contract, so I didn’t include it